zingolib/src/wallet/notes.rs

```diff
@@ -28,3 +28,22 @@ impl std::fmt::Display for NoteRecordIdentifier {
         )
     }
 }
+
+#[cfg(any(feature = "test", feature = "test-features"))]
```

Comment:
I have just realized that this is `cfg(feature = "test")`, and not `cfg(test)`. Don't like this name for that reason, but it's beyond the scope of this PR to fix. However, feature "test" relies on feature "test-features", so cfg(feature = "test-features") is sufficient here.

Reply:
Don't worry about this, I've addressed it in #951